packages/api/src/users/users.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface UserDTO {
```

Comment:
We have a `shared/src/models` directory to define models. This way they could be shared between many apps